macaroonbakery/checker.py

```diff
@@ -374,19 +349,13 @@ def _check_conditions(self, ctx, op, conds):
 
 class AuthInfo(namedtuple('AuthInfo', 'identity macaroons')):
     '''AuthInfo information about an authorization decision.
+    identity: holds information on the authenticated user as
```

Comment:
Maybe some white space between the param descriptions? I had to look a couple of times to find the different params.